allow for custom checker in getValidOutgoingTransitions

ndurell opened the pull request:
sometimes when strict is False we still want to raise some exceptions

kedder commented on src/zope/wfmc/process.py:
```diff
@@ -920,14 +934,7 @@ def getValidOutgoingTransitions(process, activity_definition,
-            else:
-                active = False
-        if active:
+        if checker(transition, process, strict):
```
I suggest to remove `process` from the checker interface, leave it just `checker(transition)`. Process is available to caller anyway. Have a checker that does not deal with exception handler at all in zope.wfmc; remove that `strict` parameter completely, it doesn't make sense anymore: client applications can have their own checkers that handle exceptions if they need.

Commit:
allow for custom checker in getValidOutgoingTransitions

sometimes when strict is False we still want to raise some exceptions
removes strict parameter

kedder commented:
:+1:

Commit:
bump version
